.github/actions/draft-release/src/utils/make_release.js

```diff
@@ -0,0 +1,53 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { RequestError } = require('@octokit/request-error');
+
+const makeRelease = async (tagName, body, commitish) => {
+	const { owner, repo } = github.context.repo;
+	const octokit = github.getOctokit(process.env.GITHUB_TOKEN);
+
+	let release;
+	try {
+		release = await octokit.rest.repos.getReleaseByTag({
+			owner,
+			repo,
+			tag: tagName
+		});
+	} catch (error) {
+		if (error instanceof RequestError && error.status === 404) {
+			release = null;
+		} else {
+			throw error;
+		}
+	}
+
+	let releaseResponse;
+
+	if (!release) {
+		releaseResponse = await octokit.rest.repos.createRelease({
+			owner,
+			repo,
+			tag_name: tagName,
+			name: tagName,
+			body,
+			draft: true,
+			prerelease: false,
+			target_commitish: commitish
+		});
+		core.info(`Created ${tagName} release`);
+	} else {
```

Comment:
This should check to see if `release` is a `draft`, and if not, throw an error. We don't want to accidentally update a previous release.

Reply:
Done in 5b6e433b6a674110dcfcc1f4f945073d0be86e31.